asar-teensy-firmware/TestScripts/Motors/TestScript1/MOTOR.h

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Remove a copy of this file and make a symlink.

Reply:
changed to symlink